install-rust-toolchain.sh

```diff
@@ -245,6 +283,11 @@ function clear_download_cache() {
 
   echo " - ${LLVM_FILE}"
   rm -f "${LLVM_FILE}"
+
+  if [ -z "${ESP_IDF_VERSION}" ]; then
+    echo " - *-elf-gcc8_4_0-esp-2021r2-patch3-linux-amd64.tar.gz"
```

Comment:
I recommend making it more generic so that we do not change the script with every patch of GCC. `*-elf-gcc*.tar.gz` should be sufficient